extras/python/audio2mozzi.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+"""
+For converting raw audio files to Mozzi table
```

Comment:
Maybe add some copyright info (you or Mozzi team and you depending on how much you drew from the original script) and a license (LGPL and GPL are already in Mozzi)?

Reply:
Done!